### DISCUSSION TITLE
TestFlight access

### DISCUSSION THREAD

**Discussion by schwa** [schwa/MastodonAgain#41] [General]
I'm looking for a handful of folks interested in helping out with testing and more importantly providing feedback and suggestions.

MastodonAgain (working name) is still **_very alpha_** but a lot of functionality is there now. Focus is primary on building a solid macOS application but due to SwiftUI the app is really coming together well on iOS.

Reply here or contact me via Mastodon private messages (or via one of the contact methods in my Mastodon profile): https://mastodon.social/@schwa

**Comment by Mattro** [comment]
I'd love to test it out. I've been using the web version so far. Happy to provide feedback & ideas too.

I used to use Twitterrific on desktop.

**Comment by jrtibbetts** [comment]
I'm game. @nrith@mas.to

**Comment by mauricerkelly** [comment]
Lemme at em! What's your preferred feedback channel; TestFlight or GH issues? If the former, sign me up; if the latter, I'll just grab source.

**Comment by schwa** [comment]
Thanks folks. Preferred feedback is via GitHub issues or discussion.

The project is still very much develop quality. A lot of features are missing I'm going to try organising https://github.com/users/schwa/projects/6/views/1 so that it's easy to see the big missing features.

As such I'm not really looking for testers at this stage, but more invested users who can provide feedback on design and features. My list of "must-have" features is going to look different from everyone else's.

It's probably going to be a few more days until I'm ready open up to a handful of people. I'll let you know. (Maybe make a mastodon list to contact folks).
